api/symfony.lock

```diff
@@ -245,18 +257,27 @@
             "ref": "471aed0fbf5620b8d7f92b7a5ebbbf6c0945c27a"
         }
     },
+    "sebastian/code-unit-reverse-lookup": {
+        "version": "1.0.1"
+    },
     "sebastian/comparator": {
         "version": "3.0.2"
     },
     "sebastian/diff": {
         "version": "3.0.2"
     },
+    "sebastian/environment": {
+        "version": "4.0.2"
+    },
     "sebastian/exporter": {
         "version": "3.1.0"
     },
     "sebastian/recursion-context": {
         "version": "3.0.0"
     },
+    "sebastian/version": {
+        "version": "2.0.1"
+    },
     "sensiolabs/security-checker": {
         "version": "4.0",
```

Comment:
composer json is "sensiolabs/security-checker": "^5.0" so you should update the composer.lock and commit it maybe ?

Reply:
This is the version of the recipe, not the package version. Check: https://github.com/symfony/recipes/tree/master/sensiolabs/security-checker/4.0